shared/src/main/scala/squants/SVector.scala

```diff
@@ -80,7 +80,7 @@ trait SVector[A] {
    * @return
    */
   def times(that: Double): SVectorType
-  def * = times _
+  def *(that: Double)  = times(that)
```

Comment:
This fails to compile in dotty. I'm not sure if what I'm doing is idiomatic

Reply:
Looks good to me.